app/components/projects/row_component.rb

```diff
@@ -57,8 +58,14 @@ def custom_field_column(column)
       cf = custom_field(column)
       custom_value = project.formatted_custom_value_for(cf)
 
-      if cf.field_format == 'text'
-        custom_value.html_safe # rubocop:disable Rails/OutputSafety
+      if cf.field_format == 'text' && custom_value.present?
+
+        concat content_tag :p, Nokogiri::HTML(custom_value).text, class: 'project-long-text', id: "#{project.id}-cf-#{cf.id}"
```

Comment:
In the case of a macro being the only content of the field, e.g. an embedded work package table no content is currently displayed. So the user has no possibility to then click on the link to display the table in the Dialog. Granted, I don't know how important doing that actually is. But we have a bug report stating that to be a problem of the former implementation (related to project description) so at least someone used it. 

Would it be possible to, similar like the check for the table, check if there is a macro in there in display the expand button same as done for the tables in this case and some placeholder text within the table?